front/src/common/api/osrdEditoastApi.ts

```diff
@@ -1800,48 +1796,116 @@ export type ScenarioPatchForm = {
   name?: string | null;
   tags?: string[] | null;
 };
-export type Comfort = 'AC' | 'HEATING' | 'STANDARD';
+export type RollingStockComfortType = 'STANDARD' | 'AC' | 'HEATING';
+export type EffortCurveConditions = {
```

Comment:
These fields are probably never undefined

Reply:
https://github.com/osrd-project/osrd/blob/dev/editoast/src/schema/rolling_stock/mod.rs#L107-L110 They are all optional here, should I override the schema to make them non optional?